css-three-column-layout/styles.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  width: 1400px;
+  height: 600px;
+  margin: 0 auto;
+  background-color: rgb(25, 150, 218);
+}
+
+.container-2 {
+  width: 1000px;
+  margin: 0 auto;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
```

Comment:
Your `h1` does not have any children, so shouldn't be doing anything with `flex` here. If you just want to center it, then do `text-align: center`